components/header/Header.vue

```diff
@@ -172,10 +199,26 @@ export default {
         }
       },
       immediate: true
-    }
+    },
+
+    /**
+     * Watches scrollingState in vuex to disable scrolling
+     * on body based on mobile header functionality
+     */
+    scrollingState: {
```

Comment:
Modifying the body style negates the use of Vuex. This could be done when opening the mobile menu instead. I would recommend adding a class to `layouts/default.vue` that is computed by this Vuex state.

Not using a class also doesn't give us the ability to use media queries for this, which is needed to prevent this from happening:
![localhost_3000_data_searchType=datasets searchTerms=(Laptop with HiDPI screen)](https://user-images.githubusercontent.com/1493195/73085050-123c8d80-3e9c-11ea-9b5a-ff5e02d9335e.png)

Steps to reproduce:
- Make viewport small enough to show the mobile header
- Open mobile menu
- Make viewport large enough to show desktop header

Reply:
Great suggestion. I don't know why I didn't think about using Vuex in layouts 🤦‍♀ I'll make the change.

Reply:
@cameronbaney made fixes as requested